apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobByPatientIdPaged.json

```diff
@@ -4896,17 +4896,17 @@
             "value" : 1
           }
         }, {
-            "url" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cnt",
-            "valueCoding" : {
-              "system" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cnt",
-              "code" : "3"
-            }
-          }, {
+          "url" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cnt",
+          "valueCoding" : {
+            "system" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cnt",
+            "code" : "3"
+          }
+        }, {
           "url" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cd",
           "valueCoding" : {
             "system" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cd",
             "code" : "3",
-            "display":"Services"
+            "display" : "Services"
```

Comment:
more curious than anything...does the new HAPI library cause these spacing nits?

Reply:
yes, the new hapi caused it but it is more consistent now so I think it is a good thing

Reply:
actually it was not the new hapi. the old hapi will generate consistently spaced files (I factored out one of the changes from this PR into another and it has these spacing changes). I'm guessing that some changes have been made by hand rather than letting the responses be generated. In any case more consistent formatting is preferable.